config/tests/jobs/jobs_test.go

```diff
@@ -293,160 +293,34 @@ func TestCommunityJobs(t *testing.T) {
 	const legacyDefault = "default"
 	const legacyTrusted = "test-infra-trusted"
 
-	// TODO: cut this list down to none
-	// this is a snapshot of docs/job-migration-todo.dm
-	// we are not overcomplicating this as all of them will be removed in august anyhow
-	// tail -n +10 docs/job-migration-todo.md | cut -d \| -f 4 | awk '{ print "\""$0"\","}'
+	// These jobs are used to automate prow itself and will not be migrating
+	// They run in prow's own cluster and will be removed right before we migrate the control plane
+	// DO NOT ADD ANY MORE JOBS HERE
 	knownBadJobs := sets.NewString(
-		"capz-azure-disk-master",
-		"capz-azure-disk-vmss-master",
-		"capz-azure-file-master",
-		"capz-azure-file-vmss-master",
-		"capz-conformance-dual-stack-master",
-		"capz-conformance-ipv6-master",
-		"capz-conformance-master",
 		"ci-test-infra-gencred-refresh-kubeconfig",
 		"ci-test-infra-rotate-legacy-default-build-sa-json-key",
```

Comment:
we should be able to drop this one next, but I want to have the cluster actually not running jobs first and then confirm

Reply:
talked to cole about this, we can clean it up.

Reply:
https://github.com/kubernetes/test-infra/pull/33274